packages/compiler/test/server/completion.ts

```diff
@@ -33,6 +33,53 @@ describe("compiler: server: completion", () => {
     ]);
   });
 
+  it("completes imports", async () => {
+    const completions = await complete(` import "┆ `, undefined, {
+      "package.json": JSON.stringify({
+        dependencies: {
+          "@cadl-lang/library1": "~0.1.0",
+          noncadllibrary: "~0.1.0",
+          nonPackageJsonLibrary: "~0.1.0",
+        },
+      }),
+      "node_modules/@cadl-lang/library1/package.json": JSON.stringify({
+        cadlMain: "main.cadl",
+      }),
+      "node_modules/noncadllibrary/package.json": JSON.stringify({}),
```

Comment:
can you put a `main: "./foo.js"` to represent more of a regular library

Reply:
committed this changed